tools/c-parser/RELEASES.md

```diff
@@ -169,3 +169,20 @@
 - Updated SIMPL from the AFP
 - Ensure that umm_types.txt is saved relative to theory file
 - If cpp_path is relative, make it relative to the current theory
+
+
+## 1.22
+
+- Builds with Isabelle2025
+- Makefile: fail early if L4V_ARCH is not set
+- Suppress internal simpset warnings in Simpl vcg
+- add more documentation in README
+- do not export Separation Logic notation by default;
+  Separation Logic can be enabled manually using the command
+
+    unbundle C_seplog_syntax
+
+-  Add the option "no_modifies" to the install_C_file command to skip
+   the generation of modifies proofs.
```

Comment:
"Add the option" took me a bit to grasp. Checking: this version adds an option one can use to skip the modifies proofs, and not that it adds the option as default to the install_C_file command?

style variation: Makefile has to start with a cap, for the rest suggest picking cap or no cap
also, these two lines have extra indent